test/e2e/e2e_test.go

```diff
@@ -291,7 +292,7 @@ var _ = ginkgo.Describe("JobSet", func() {
 			// Every ReplicatedJob runs 1 container to sleep for 10 seconds.
 			rJobModelInitializer := dependsOnTestReplicatedJob(ns, modelInitializerJob, numReplicas, nil, nil)
 			rJobDatasetInitializer := dependsOnTestReplicatedJob(ns, datasetInitializerJob, numReplicas, nil, nil)
-			rJobTrainer := dependsOnTestReplicatedJob(ns, trainerJob, numReplicas, nil,
+			rJobTrainer := dependsOnTestReplicatedJobWithSleepDuration(ns, trainerJob, numReplicas, nil, 0,
```

Comment:
If we don't have sleep, how we can make sure that ReplicatedJobs are executed in order ?

Reply:
This is only about the trainer jobs, i.e the jobs no other jobs depend on. The tests are written so that you start the dependencies, then check only they are started, then wait for all the jobs to exist and the jobset to complete. Nobody really checks the status of the trainer jobs.

Reply:
Oh, that sounds good!